moved cdk depencencies into devDependencies (#29)
nearform/mira: docs/README.md

 * A Command Line Interface, that helps to initialise and deploy applications, and also to deploy a CI pipeline.
 * Foundational constructs that simplify configuration of multiple application environments.
 * An opinionated set of AWS CDK aspects, e.g. IAM Policy validator. 
 * Complementary sample applications e.g. a simple S3-based web hosting app.
 
__Note: Currently supported version of the AWS CDK is: *[1.49.1](https://github.com/aws/aws-cdk/releases/tag/v1.49.1)*.__
 

> Please make sure you application will depend on that exact version.

Reviewer: ```suggestion
Please make sure your application depends on that exact version of the CDK.
```
